### PR TITLE
Add flash notice when balance is not enough

### PR DESCRIPTION
**Pull request by eordano**
image

### CODE REVIEW

**Comment by nicosantangelo** [comment]
I found a few things testing it out:

- It doesn't seem to work if my balance is `0`
- The `x` on the notification doesn't have a `cursor pointer`
- The notification rectangle is on top of the zoom controls
- If I click in two parcels without waiting for the notification to go away, the time it's displayed isn't consistent (it disappears quickly). **I don't think this is a dealbreaker**

notif

**Comment by nicosantangelo** [comment]
- If I use the app with Metamask locked I am redirected to `http://localhost:3000/walletError` (_expected_) but unlocking and pressing the Decentraland logo leads me to a perpetual loading indicator. **This is not happening on master**, it's due to the removal of `connectWeb3` on `HomePageContainer`
- Furthermore, entering `http://localhost:3000/unsubscribe` fires `connectWeb3` twice, as it wasn't removed there and I managed to do this visiting from there:

image

(_again, cannot reproduce in master_)

**Comment by nicosantangelo** [comment]
- **Not a dealbreaker**: I'm a bit lost with the implementation. Why not a `FalshContainer` that handles the timeout that's now found on utils (with React in it) and adding that to the `HomePage` component ?
The modal was added in this way and sits on top of the other components when rendered. Further, it's _almost_ the same as adding another HTML element on `index.html` but without escaping React

**Comment by nicosantangelo** [comment]
- **Not a huge deal**: the notification it's on top of the modals and looks a bit weird:
image

File: webapp/src/sagas.js [hunk 1]
```diff
@@ -408,7 +410,7 @@ function getBidGroupsNonce(bidGroups) {
 function* handleFastBid(action) {
   const parcel = action.parcel
   if (parcel.projectId) {
-    // TODO: Show message, can't bid on project
+    yield call(() => flashNotice(`You can't bid on a project parcel`))
```
Comment (nicosantangelo) [inline webapp/src/sagas.js]: This code is never reached as we are checking for:

image

before bidding. That said I don't think it's a bad idea to have it here :)

Comment (eordano) [reply webapp/src/sagas.js]: It's a side effect! flashNotice has to be here --that's what sagas are for

**Comment by eordano** [comment]
Thanks for the comments, going through them now

**Pull request merged**